actions/notification_actions.jsx

```diff
@@ -115,6 +115,10 @@ export function sendDesktopNotification(post, msgProps) {
             }
         }
 
+        if (isCrtReply) {
+            title = Utils.localizeMessage('notification.crt', 'Reply in ') + title;
```

Comment:
Concatenating the title onto `Reply in` might not work depending on the language. We can use `Utils.localizeAndFormatMessage` to let translators order the title the way they want